src/config.ts

```diff
@@ -0,0 +1,17 @@
+import {ClientOptions, Intents} from 'discord.js';
```

Comment:
Can we get a high level overview of what this feature does? I am wondering what the purpose of this file is

Reply:
It's the config file for the bot, so that if any bot options need to be changed we don't have to make changes to index.ts

Reply:
I mean in a comment lol